### DISCUSSION TITLE
MC Ultimate - Powered by Sinytra!

### DISCUSSION THREAD

**Discussion by Cixon** [Sinytra/Connector#110] [Show and tell]
Check out my modpack guys! Goal is similar to most V+ packs.
Perhaps we can get this up there next to the big leagues like Better MC? 🤔

 
![IMG_8333](https://github.com/Sinytra/Connector/assets/69068153/bd76186a-a1be-4d03-85f2-49303c67b1ee)

Link
https://legacy.curseforge.com/minecraft/modpacks/ult

**Comment by jLn0n** [comment]
can u also upload that to modrinth or link it?
